### DISCUSSION TITLE
Welcome to zcad Discussions!

### DISCUSSION THREAD

**Discussion by zamtmn** [zamtmn/zcad#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by petuhoff** [comment]
Добрый день очень хотелось бы с вами пообщатся на тему данного проекта. Я директор ООО "ЗВС" мы разработчики ПО SimInTech. https://simintech.ru  это конкурент Simulink написана на Deдphi. У нас есть модуль расчета электрических систем. сейчас есть заказчик который хочет кроме расчета и иметь еще подготовку проекта. Вполне возможно взаимовыгоное сотрудниество. Хотелось бы с вам связаться мой адресс petuhoff@mail.ru
